monai/inferers/utils.py

```diff
@@ -300,6 +300,7 @@ def sliding_window_inference(
 
     # remove padding if image_size smaller than roi_size
     if any(pad_size):
+        kwargs.update({"pad_size": pad_size})
```

Comment:
May I know the reason for this update?

Reply:
Removed the padding within slidingwindow inferer and moved all padding to vista3d forward function. This pad_size need to be passed to vista3d.